TFT/src/User/Hal/LCD_Encoder.c

```diff
@@ -4,21 +4,19 @@
 
 #if ENC_ACTIVE_SIGNAL
 
-void LCD_Enc_InitActiveSignal(bool marlinType_LCD12864)
```

Comment:
all the files in HAL folder have been made independent from infoSettings, machineSettings etc.... For that reason, that param was passed as an argument.

Reply:
Ok noted

Reply:
thanks for the changes. Now it's also better than the old version